### DISCUSSION TITLE
NextJS with Tailwind does not work when importing from Global.css

### DISCUSSION THREAD

**Discussion by robertwt7** [vercel/next.js#25845] [Help]
### What version of Next.js are you using?

10.2.3

### What version of Node.js are you using?

v12.20.0

### What browser are you using?

Chrome and Edge

### What operating system are you using?

Windows

### How are you deploying your application?

yarn dev

### Describe the Bug

Following the setup [here](https://tailwindcss.com/docs/guides/nextjs) in tailwind, if I choose to import tailwind from `global.css` by putting these directives:

```
/* ./styles/globals.css */
@tailwind base;
@tailwind components;
@tailwind utilities;

```

It doesn't work. 

However, if i import tailwind straightaway as shown in the [example](https://github.com/vercel/next.js/blob/canary/examples/with-tailwindcss/tailwind.config.js#L2) from nextjs, then it works.

Currently here's the way I import it in `_app.js`:
`import 'tailwindcss/tailwind.css';`

Not sure if this is a tailwind bug or nextjs bug, sorry if i mistakenly reported a bug here

### Expected Behavior

Tailwind css should be compiled when running `yarn dev`

### To Reproduce

Exactly following [this](https://tailwindcss.com/docs/guides/nextjs) step from tailwind tutorial with the second option when including css (using @tailwind directives)

## Edit:

Nevermind this, somehow I tried creating new project and it works. Probably just some random stuff, or maybe even restarting the server should've worked

**Comment by wwwdepot** [comment]
The [tailwind example](https://github.com/vercel/next.js/tree/canary/examples/with-tailwindcss) works fine when I switch to a custom css file.

```
.
├── package.json
├── pages
│   ├── api
│   │   └── hello.js
│   ├── _app.js
│   └── index.js
├── postcss.config.js
├── README.md
├── styles
│   └── global.css
├── tailwind.config.js
└── yarn.lock
```

### `_app.js`

```js
//import 'tailwindcss/tailwind.css'
import "../styles/global.css";

function MyApp({ Component, pageProps }) {
  return <Component {...pageProps} />
}

export default MyApp
```
### `./styles/global.css`

```css
@tailwind base;
@tailwind components;
@tailwind utilities;
```

**Reply by robertwt7** [reply]
huh now mine doesn't even work with the jit. i'm using lerna, not sure if that affects the project situation

**Comment by kyimoemin** [comment]
I do as the @wwwdepot said and it works but you need to delete node_modules and run `npm i` again.

**Comment by alfredzh** [comment]
Same here, @wwwdepot 's solution works fine. Thank you. Hope this will be fixed.
